src/components/BorrowerProfile/BorrowerSideSheetContent.vue

```diff
@@ -0,0 +1,155 @@
+<template>
```

Comment:
This component is looking much different from the one in CPS and missing content. One of the related CPS tickets was linked in Jira. You can see the [CPS side sheet for BP live currently on dev](https://www.development.kiva.org/lend-category-beta). We should be able to port the components straight from CPS.